src/corner-kick/README.md

```diff
@@ -0,0 +1,57 @@
+# Corner Kick
+
+Corner Kick is an Electron app that allows for the visualization of UBC Thunderbots' simulator.
+
+## Contribute
+
+### Node
+
+You will need Node to contribute to this project. To install Node, run the following:
+
+```
+sudo apt-get install nodejs npm
+```
+
+### Yarn
+
+Yarn is considered a more powerful package manager than npm. You should install it by running:
+
+```
+npm install -g yarn
+```
+
+### Installing the project
+
+With Node installed, clone the project and install the dependencies.
+
+```
+yarn install
+```
+
+### Running the dev server
+
+You can test your code by running:
+
+```
+yarn start
+```
+
+A window should open with the application in development mode.
```

Comment:
How do we toggle between `development` and `release`? It would be awesome if `catkin_make` built this in `release`.

Reply:
Done. `catkin_make` will now build this in release.